src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -178,13 +212,18 @@ function _parseTabAttributes(element) {
  */
 
 function _parseBoxAttributes(element) {
+  _warnInconsistentAttributes(['light', 'seamless'], element.attribs);
+  _warnInconsistentAttributes(['no-background', 'background-color'], element.attribs);
+  _warnInconsistentAttributes(['no-background', 'seamless'], element.attribs);
+  _warnInconsistentAttributes(['no-border', 'border-color'], element.attribs);
+  _warnInconsistentAttributes(['no-border', 'border-left-color'], element.attribs);
+  _warnInconsistentAttributes(['no-border', 'seamless'], element.attribs);
+  _warnInconsistentAttributes(['no-icon', 'icon'], element.attribs);
```

Comment:
[LFC] 
I considered changing 
```
function _warnInconsistentAttributes(inconsistentAttribs, elemAttribs) {
  const hasInconsistentAttribs = inconsistentAttribs.every(
    inconsistentAttrib => inconsistentAttrib in elemAttribs);

  if (hasInconsistentAttribs) {
    // TODO: Use logger here instead of console.log. See issue #1060
    // logger.warn(`Usage of inconsistent attributes: ${inconsistentAttribs.join(', ')}`);
    // eslint-disable-next-line no-console
    console.log(`warn: Usage of inconsistent attributes: ${inconsistentAttribs.join(', ')}`);
  }
}
```